### PR TITLE
fix(docker): Enable build on schedule event

### PR DESCRIPTION
**Pull request by heliocastro**
After move to scheduled build, main images are building only on push and workflow_dispatch events.
Noticed after some mistake PR done on Ort Github CI repository.

### CODE REVIEW

File: .github/workflows/docker-ort-runtime-ext.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ name: Docker extended runtime image
 on:
   workflow_dispatch:
   schedule:
-    - cron: '0 4 * * *'
+    - cron: '0 5 * * *'
```
Comment (fviernau) [inline .github/workflows/docker-ort-runtime-ext.yml:23]: This change IMO should be a separate commit. e.g. "align job dispatch time of ort-runtime-ext and ort-runtime"

**Review by fviernau — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-ort-runtime-ext.yml (hunk 1 above)
Comment (fviernau) [reply .github/workflows/docker-ort-runtime-ext.yml:23]: For the commit which re-enables it, IMO it would be nice to have a reference to the commit which accidentally disabled it.

Comment (heliocastro) [reply .github/workflows/docker-ort-runtime-ext.yml:23]: @fviernau We never disabled. Actually, we moved from push events to scheduled events.
And after we moved, the schedule event were never handled.

Comment (heliocastro) [reply .github/workflows/docker-ort-runtime-ext.yml:23]: @fviernau Splitted in two commits and commented about previous commit. 
The next window to build will be 4 AM and 5 AM.

**Review by tsteenbe — APPROVED** [review]

**Review by fviernau — APPROVED** [review]

**Comment by heliocastro** [comment]
@fviernau @tsteenbe @sschuberth Anyone can push this one before tomorrow 4AM schedule ?

**Comment by heliocastro** [comment]
@fviernau The pr is clean, can we go with approval now ?

**Review by tsteenbe — APPROVED** [review]
